semver/greater_than_range.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+import type { Range, SemVer, SemVerRange } from "./types.ts";
+import { rangeMax } from "./range_max.ts";
+import { greaterThan } from "./greater_than.ts";
+
+/** Checks to see if the version is greater than all possible versions of the range. */
```

Comment:
If one asks, "Is this version greater than the range?" perhaps it's worth swapping the argument order. Ditto with the other new functions. What do we think?

Reply:
My train of thought was to pass the context element first, e.g. the range. But that makes a lot of sense.